scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -367,6 +369,21 @@ class ScioContext private[scio] (val options: PipelineOptions,
   }
 
   /**
+   * Get an SCollection for a Protobuf file.
+   * @group input
+   */
+  def protobufFile[T: ClassTag](path: String)
+                               (implicit ev: T <:< GeneratedMessage with Message): SCollection[T] =
```

Comment:
`GeneratedMessage` already implements `Message` right? Is `with Message` necessary here?